src/datatypes/user/User.ts

```diff
@@ -16,8 +16,8 @@ export default class User implements LoginCredentials {
   password: string; // Hashed Password
   memberSince: Date;
   admissionYear: number;
-  nameKorean: string;
-  nameEnglish?: string | null; // null for DB
+  legalName: string;
+  nickname: string | null; // null for DB
```

Comment:
As `nickname` field allow null, the `nickname` field in `src/datatypes/user/NewUserForm.ts` should be optional